.github/workflows/coverage.yml

```diff
@@ -0,0 +1,31 @@
+name: Running Code Coverage
+
+on: [push, pull_request]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [14.x]
```

Comment:
Honestly, not a fan of the Matrix strategy if you’re not doing anything matrix-y.

Reply:
@sjetha Hi, can you check now? I have modified the PR